src/System-SessionManager/WorkingSession.class.st

```diff
@@ -64,7 +58,6 @@ WorkingSession >> executeDeferredStartupActions: resuming [
 WorkingSession >> initialize [
 	super initialize.
 	deferredStartupActions := OrderedCollection new.
-	creationTime := DateAndTime now.
```

Comment:
I wondered if this was used. 
In the redefinition later will we reintroduce it?

Reply:
Probably for sorting them.

Reply:
Nope, not even...